flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -718,7 +718,7 @@ class OmpAttributeVisitor : DirectiveAttributeVisitor<llvm::omp::Directive> {
   void CheckDataCopyingClause(
       const parser::Name &, const Symbol &, Symbol::Flag);
   void CheckAssocLoopLevel(std::int64_t level, const parser::OmpClause *clause);
-  void CheckObjectInNamelistOrAssociate(
+  void CheckObjectIsPrivatisable(
```

Comment:
I've noticed that sometimes `privatisable` (British English) is used and other times it's `privatizable`.
It would be nice to standardize one of the two in the code.
I suggest the latter, because that's how it's written in the OpenMP standard, that uses words such as `privatization` and `privatized`.

Reply:
I agree we should use the American spelling.